imports/plugins/included/payments-authnet/server/methods/authnet.js

```diff
@@ -62,12 +62,16 @@ const ValidCVV = Match.Where(function (x) {
 Meteor.methods({
   authnetSubmit: function (transactionType = "authorizeTransaction", cardInfo, paymentInfo) {
     check(transactionType, String);
-    check(cardInfo, {
-      cardNumber: ValidCardNumber,
-      expirationYear: ValidExpireYear,
-      expirationMonth: ValidExpireMonth,
-      cvv2: ValidCVV
-    });
+    try {
+      check(cardInfo, {
+        cardNumber: ValidCardNumber,
+        expirationYear: ValidExpireYear,
+        expirationMonth: ValidExpireMonth,
+        cvv2: ValidCVV
+      });
+    } catch (error) {
+      throw new Meteor.Error(400, "Invalid card details");
```

Comment:
So passing in this number here is deprecated. Please check the style guide but basically it should be in the format
`Meteor.Error("short-description", "Detailed description")`